src/shell/components/FontAwesomeIcon.tsx

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import React = require('react');
+
+export class FontAwesomeIcon extends React.Component<{ path?: string }, {}> {
```

Comment:
There are some assumptions related to the paths of FontAwesome SVGs (such as a specific viewBox) that might not apply to our own, custom icons and that is why I created a specific component for FontAwesome icons.